manifests/02-reviewsv3.yaml

```diff
@@ -3,6 +3,34 @@
 ##################################################################################################
 apiVersion: extensions/v1beta1
 kind: Deployment
+metadata:
+  name: reviews-v2
```

Comment:
I am wondering why we need `reviews-v2` deployment in this file. It's defined in the previous manifest https://github.com/pavolloffay/istio-bookinfo-redhat-forum-zurich-2019/blob/master/manifests/01-reviewsv2-ratings.yaml#L48

Reply:
I will remove it in the follow-up commit.